drivers/scheduler/k8s/k8s.go

```diff
@@ -662,6 +663,15 @@ func (k *K8s) filterPureVolumesIfEnabled(claim *v1.PersistentVolumeClaim) (bool,
 	return k.filterPureTypeVolumeIfEnabled(claim, volTypes)
 }
 
+// filterPureVolumesIfEnabled will return whether or not the given PVC is a Pure
+// direct access volume, only if PureVolumes is true. If PureVolumes is false, all
+// volumes are valid.
+func (k *K8s) filterPureVolumesIfEnabledByType(claim *v1.PersistentVolumeClaim, pureVolType string) (bool, error) {
```

Comment:
Just to clarify, what is a "type" here? Storage class name? `backend` param value? Might want to rename it so it's a bit more clear

Reply:
Seems like it's "PureFile" or "PureBlock", can we call it `pureVolBackend` then?

Reply:
Bump on this